src/main/java/bio/terra/cli/serialization/userfacing/resource/UFBqTable.java

```diff
@@ -57,12 +56,13 @@ public void print(String prefix) {
     super.print(prefix);
     PrintStream OUT = UserIO.getOut();
     OUT.println(prefix + "GCP project id: " + projectId);
-    OUT.println(prefix + "BigQuery dataset id: " + datasetId + " table id: " + dataTableId);
+    OUT.println(prefix + "BigQuery dataset id: " + datasetId);
+    OUT.println(prefix + "BigQuery table id: " + dataTableId);
 
     if (tableDescription != null) {
-      OUT.println(prefix + "table description: " + tableDescription);
+      OUT.println(prefix + "Table description: " + tableDescription);
     }
-    OUT.println(prefix + "table has row #: " + numRows.toString());
+    OUT.println(prefix + "# Rows: " + numRows.toString());
```

Comment:
I think we should have nice translations of `null` here (e.g. `(undefined)` or `(unknown)`). You can see an example in the other `UF` resource class for BQ datasets